config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
What do you think about removing this file? I think it's only for ActionCable? If we want to add ActionCable, we can add this file then :)